qa/integration/services/elasticsearch_setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+current_dir="$(dirname "$0")"
+source "${current_dir}/helpers.sh"
+
+if [ -n "${ES_VERSION+1}" ]; then
+  echo "Elasticsearch version is $ES_VERSION"
+  version=$ES_VERSION
+else
+   version=5.0.0-alpha5
+fi
+
+setup_es $version
+start_es
```

Comment:
The **setup_es**, **start_es** seems to be really specific to this file and are defined in the helpers.sh?
It might be a good idea to actually define them where they are reused?